src/components/SystemMessageNotification/SystemMessageNotification.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-bootstrap';
+import styles from './SystemMessageNotification.module.scss';
+
+
+export function getSystemMessageType(type: string) {
```

Comment:
Map the original SystemMessageNotificationTypes over, not a string. This is susceptible to mistakes in the future.

Reply:
@jasekiw It throws an  error S2345: Argument of type 'string' is not assignable to parameter of type 'SystemMessageNotificationTypes'.